lib/renderer/inspector.ts

```diff
@@ -36,9 +52,18 @@ const createMenu = function (x: number, y: number, items: ContextMenuItem[]) {
   const isEditMenu = useEditMenuItems(x, y, items);
   ipcRendererInternal.invoke<number>(IPC_MESSAGES.INSPECTOR_CONTEXT_MENU, items, isEditMenu).then(id => {
     if (typeof id === 'number') {
-      window.DevToolsAPI!.contextMenuItemSelected(id);
+      if (contextIsolationEnabled) {
+        webFrame.executeJavaScript(`window.DevToolsAPI.contextMenuItemSelected(${JSON.stringify(id)})`);
+      } else {
+        window.DevToolsAPI!.contextMenuItemSelected(id);
+      }
```

Comment:
I think the `webFrame.executeJavaScript` version works fine regardless of the value of `contextIsolationEnabled`, right?

Possibly `internalContextBridge.overrideGlobalValueFromIsolatedWorld` works fine in non-contextIsolation situations too, and we can drop all these conditionals.

Reply:
@nornagon i'm happy with either approach, though i took this one given security hazards of `webFrame.executeJavaScript` and the fact that we're needing to do string templating in there

Reply:
If there are security hazards, we shouldn't be using it at all, regardless of whether the context is isolated :) What you have there seems OK to me, what sort of hazard are you thinking of?

Reply:
just the general risk of executing js with templated strings  - you're right though, it's fine enough and i'll make the change!